### ISSUE TITLE
api.test_permission.PermissionTestCase.test_positive_search_by_resource_type - some changes detected in 6.3

### ISSUE THREAD

**Issue by rplevka** [SatelliteQE/robottelo#4130]
```
tests/foreman/api/test_permission.py:126: in test_positive_search_by_resource_type
    self.fail(json.dumps(failures, indent=True, sort_keys=True))
E   AssertionError: {
E    "Host": {
E     "added": [
E      "play_roles"
E     ]
E    }
E   }
```
related to https://github.com/SatelliteQE/robottelo/issues/4073

**Comment by omaciel** [comment]
Is this something for you @jyejare ?

**Comment by jyejare** [comment]
@omaciel , This is not something related to Canned Role feature. All the tests for canned role has been added to test_role.py file. But its really a good candidate for Canned Role feature.

**Comment by rplevka** [comment]
This is related to Ansible feature

**Comment by omaciel** [comment]
Thank you @jyejare and @rplevka  for the clarification. Should we ask @rochacbruno to see if he has the bandwidth to work on this or ask @abalakh or @oshtaier  ?

**Comment by oshtaier** [comment]
@omaciel 
Already fixed in https://github.com/SatelliteQE/robottelo/pull/4122

**Comment by omaciel** [comment]
@oshtaier awesome! Do we close this issue then?

**Comment by oshtaier** [comment]
Fixed in #4122

**Comment by rplevka** [comment]
@oshtaier thanks!
